tutorial/8_jittering_tutorial.ipynb

```diff
@@ -0,0 +1,351 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "8ad4167a-e4e7-498d-909a-c04da9f177ed",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Correct use of Jittering\n",
+    "This tutorial discusses the background and correct use of the jittering/dithering method implemented in the package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fb045e17-fa89-4c11-9d51-7f06e80d96d5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os\n",
+    "\n",
+    "import sed\n",
+    "\n",
+    "%matplotlib widget"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "42a6afaa-17dd-4637-ba75-a28c4ead1adf",
+   "metadata": {},
+   "source": [
+    "## Load Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "34f46d54",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_path = '.' # Put in Path to a storage of at least 20 Gbyte free space.\n",
+    "if not os.path.exists(data_path + \"/WSe2.zip\"):\n",
+    "    os.system(f\"curl -L --output {data_path}/WSe2.zip https://zenodo.org/record/6369728/files/WSe2.zip\")\n",
+    "if not os.path.isdir(data_path + \"/Scan049_1\") or not os.path.isdir(data_path + \"/energycal_2019_01_08/\"):\n",
+    "    os.system(f\"unzip -d {data_path} -o {data_path}/WSe2.zip\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f1f82054",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The Scan directory\n",
+    "fdir = data_path + '/Scan049_1'\n",
+    "# create sed processor using the config file:\n",
+    "sp = sed.SedProcessor(folder=fdir, config=\"../sed/config/mpes_example_config copy.yaml\")"
```

Comment:
I guess this should be just mpes_example_config

Reply:
Thanks, fixed